packages/squiggle-lang/src/fr/builtin.ts

```diff
@@ -69,6 +70,38 @@ export const library = [
       }),
     ],
   }),
+  maker.make({
+    name: "equal",
+    definitions: [
+      makeDefinition([frAny, frAny], ([a, b]) => {
+        if (a.toString() !== b.toString()) {
```

Comment:
It's possible that there are some rare cases where this shouldn't hold, so this is a bit risky.